### PR TITLE
[3006.x] Issue 64111

### PR DESCRIPTION
**Pull request by dwoz**
### What does this PR do?

* Add regression test for #64111

### CODE REVIEW

**Review by twangboy — DISMISSED** [review]

**Comment by twangboy** [comment]
Looks like you got some pre-commit and lint failures

File: salt/utils/templates.py [hunk 1]
```diff
@@ -515,10 +515,6 @@ def opt_jinja_env_helper(opts, optname):
         raise SaltRenderError(
             "Jinja error: {}{}".format(exc, out), line, tmplstr, trace=tracestr
         )
-    finally:
-        if loader and hasattr(loader, "_file_client"):
```
Comment (s0undt3ch) [inline salt/utils/templates.py:519]: This means the file client is left open? When is it closed?

**Comment by s0undt3ch** [comment]
Your test case is now included in https://github.com/saltstack/salt/pull/64113

**Comment by s0undt3ch** [comment]
The test case was included in https://github.com/saltstack/salt/pull/64113

**Pull request closed without merging**